test/support/stack.ex

```diff
@@ -8,7 +8,7 @@ defmodule Stack do
     Agent.update(__MODULE__, &[item | &1])
   end
 
-  def pop(item) do
+  def pop do
```

Comment:
@cjbottaro This one is odd. The `Stack` module is referenced a handful of times, but I can't find anything that really uses it. The `Pinger` middleware module references `items` but doesn't push anything onto the stack, and `Pinger` itself doesn't seem to be referenced by anything. Is this dead code?

Reply:
Yeah, I think `Pinger` and `Stack` are dead code. They were originally meant to facilitate communication between a `Faktory.Job` and tests, but that's done differently now. I vote for removing them!

Reply:
Sounds good. I should have time to finish this up tonight.